app/api/v1/endpoints/trading.py

```
@@ -448,29 +451,58 @@ async def get_portfolio_status(
-        # Get portfolio data from risk service
-        portfolio_data = await risk_service.get_portfolio_status(str(current_user.id))
+        # Get real portfolio data using existing exchange balance system
+        from app.api.v1.endpoints.exchanges import get_user_portfolio_from_exchanges
+        portfolio_data = await get_user_portfolio_from_exchanges(str(current_user.id), db)
         
         if not portfolio_data.get("success"):
             raise HTTPException(
                 status_code=status.HTTP_404_NOT_FOUND,
-                detail="Portfolio data not found"
+                detail=portfolio_data.get("error", "Portfolio data not found")
             )
```

Ruff (0.12.2)

Abstract `raise` to an inner function

(TRY301)